packages/superset-ui-chart-controls/src/sections.tsx

```diff
@@ -101,7 +101,6 @@ export const annotations = {
           default: [],
           description: 'Annotation Layers',
           renderTrigger: true,
-          tabOverride: 'data',
```

Comment:
Removing this here as it's support has been stripped in the PR on `incubator-superset` - this section will remain on the Data tab anyway.